platform/security/policy/security-policy-context/src/main/java/org/codice/ddf/security/policy/context/impl/PolicyManager.java

```diff
@@ -457,6 +468,40 @@ public boolean getSessionAccess() {
     return sessionAccess;
   }
 
+  public void setPolicyFilePath(String policyFilePath) {
+    Map<String, String> properties =
+        PropertiesLoader.getInstance()
+            .toMap(PropertiesLoader.getInstance().loadProperties(policyFilePath));
+
+    if (properties.isEmpty()) {
+      LOGGER.debug("File-based authentication type configuration not found.");
+      return;
+    }
+
+    LOGGER.info("Proceeding with file-based authentication type configuration ...");
+
+    Map<String, List<String>> contextToAuthTypes = new HashMap<>();
+
+    for (Map.Entry<String, String> authTypes : properties.entrySet()) {
+      List<String> finalAuthTypes = new ArrayList<>();
+      String[] auths = authTypes.getValue().trim().split("\\s+then\\s+");
```

Comment:
✏️ Maybe leave a comment on what `\\s+then\\s+` is doing. Cause I ain't regex literate

❓ Looks like we now have two authentication type formats:
```
PKI|SAML
```
and
```
PKI then SAML
```
Should we decide on one form and use that throughout the system?

Reply:
The file-based config can be more verbose and explicit. I don't think there is an issue with having | or then in different configs. Like I mentioned we're hoping no one uses the file-based config and we can get rid of it.